src/components/interaction-designer/toolbar/BlockErrorsExpandable.vue

```diff
@@ -0,0 +1,96 @@
+<template>
```

Comment:
I had this urge for unification myself. However, the two menus are different.
In the block errors list, we want to collapse sublists (for each block) so that we can easily scroll and navigate between the sublists and not get lost.
In the flow errors list, there is no such grouping, the list is flat. Adding a 'Show more' button wouldn't give any benefit but would lead to unnecessary effort for the user (to open the menu first and then press the 'Show more' button). In terms of limiting the list if it's too long - it gets limited already (and becomes scrollable).

Reply:
If the flow errors menu changes (e.g. we add categories to it), then we can indeed unify the components.

Reply:
Thanks @bulatgab , yeah ... for now it would be hard to unify it as you said. We can consider only block errors, I agree.